modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -107,111 +107,434 @@ Status description:: error: data exception - unsupported value. The provided val
+
+=== 22N97
+
+Status description:: error: data exception - unexpected struct tag. Unexpected struct tag: `{ $tagStringAsHex }`.
+
+=== 22N98
+
+Status description:: error: data exception - wrong first field during deserialization. Unable to deserialize request. Expected first field to be `{ %s }`, but was '{ %s }'.
+
+=== 22N99
+
+Status description:: error: data exception - wrong token during deserialization. Unable to deserialize request. Expected `{ %s }`, found `{ %s }`.
+
+=== 22NA0
+
+Status description:: error: data exception - invalid property based access control rule. Failed to administer property rule.
+
+// === 22NA1
+
+// Status description:: error: data exception - invalid property based access control rule involving non-comutative expressions. The property `{ $propKey }` must appear on the left hand side of the `{ $operation }` operator.  
+
+// === 22NA2
+
+// Status description:: error: data exception - invalid property based access control rule involving multiple properties. The expression: `{ $expr }` is not supported. Property rules can only contain one property.
+
+// === 22NA3
+
+// Status description:: error: data exception - invalid property based access control rule involving NaN. 'NaN' is not supported for property-based access control.
+
+=== 22NA4
+
+Status description:: error: data exception - invalid property based access control rule involving comparison with NULL. The property value access rule pattern `{ $pred }` always evaluates to 'NULL'.
+
+=== 22NA5
+
+Status description:: error: data exception - invalid property based access control rule involving IS NULL. The property value access rule pattern `{ $pred }` always evaluates to 'NULL'. Use `IS NULL' instead.
+
+=== 22NA6
+
+Status description:: error: data exception - invalid property based access control rule involving IS NOT NULL. The property value access rule pattern `{ $pred }` always evaluates to 'NULL'. Use 'IS NOT NULL' instead.
+
+// === 22NA7
+
+// Status description:: error: data exception - invalid property based access control rule involving nontrivial predicates. The expression: `{ $expr }` is not supported. Only single, literal-based predicate expressions are allowed for property-based access control.
+
+=== 22NA8
+
+Status description:: error: data exception - parsing JSON exception. Underlying error: `{ $cause }`.
+
+// === 22NA9
+
+// Status description:: error: data exception - unexpected map entry. Invalid input. Unexpected key `{ $key }`, expected keys are `{ $listAllowedKeys }`.
+
+// === 22NB0
+
+// Status description:: error: data exception - invalid property based access control rule involving WHERE and IS NULL. The property value access rule pattern `{ $pred }` always evaluates to 'NULL'. Use `WHERE' syntax in combination with `IS NULL` instead.
+
+// === 22NFF
+
+// Status description:: error: data exception - referent does not exist. The referent of the specified reference does not exist.
```

Comment:
It looks like this one was added by Stefan in the spreadsheet, but it is not in the codebase and not used in any of the teams' errors in the other spreadsheet. Also it has a much higher number than the other 22x errors so I get the feeling there is a reason it is there but I don't know it. Maybe we can keep it in the spreadsheet for now, but exclude it from docs until we know if it has any usages?